src/UNL/UCBCN/Event.php

```diff
@@ -114,6 +115,10 @@ public function getEditURL($calendar) {
         return Controller::$url . $calendar->shortname . '/event/' . $this->id . '/edit/';
     }
 
+    public function getAddDatetimeURL($calendar) {
+        return Controller::$url . $calendar->shortname . '/event/' . $this->id . '/datetime/add/';
```

Comment:
You could simplify this by calling `$calendar->getManageURL()` instead of replicating that logic again.